<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>point_auto_picker</class>
 <widget class="QDialog" name="point_auto_picker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Pick Points</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="max_threshold_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pixels above this threshold value will not be considered. This is largely to eliminate hot pixels (which may not be necessary if background subtraction is performed).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Max Threshold</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="threshold_group">
     <property name="title">
      <string>Min Threshold</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="threshold_slider_label">
        <property name="text">
         <string>Slider:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="threshold_slider_layout">
        <item>
         <widget class="ScientificDoubleSpinBox" name="threshold_slider_min">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimum value for the threshold slider.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="threshold_slider">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The threshold slider. Edit this to visualize changes in auto-picked points.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScientificDoubleSpinBox" name="threshold_slider_max">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximum value for the threshold slider.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>8</number>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>5000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="threshold_label">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="threshold">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The current value of the threshold slider.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="value">
         <double>2000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dilation_radius_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The radius for disk dilation before labeling spots.&lt;/p&gt;&lt;p&gt;Features in the image are dilated before labeling spots so that peaks that are very close may be combined into one peak. Setting this radius to zero disables it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Dilation Radius:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="dilation_radius">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The radius for disk dilation before labeling spots.&lt;/p&gt;&lt;p&gt;Features in the image are dilated before labeling spots so that peaks that are very close may be combined into one peak. Setting this radius to zero disables it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="max_threshold">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pixels above this threshold value will not be considered. This is largely to eliminate hot pixels (which may not be necessary if background subtraction is performed).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="value">
      <double>65535.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="min_area_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum area of labeled spots to keep. This eliminates smaller spots from the auto-selection.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Min Area:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ScientificDoubleSpinBox" name="min_area">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum area of labeled spots to keep. This eliminates smaller spots from the auto-selection.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
     <property name="value">
      <double>300.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>threshold_slider_min</tabstop>
  <tabstop>threshold_slider</tabstop>
  <tabstop>threshold_slider_max</tabstop>
  <tabstop>threshold</tabstop>
  <tabstop>max_threshold</tabstop>
  <tabstop>min_area</tabstop>
  <tabstop>dilation_radius</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>point_auto_picker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>point_auto_picker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
